Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
```

Comment:
You selected the wrong line in your comment.

That said, the -O0 is required to make sure I get the right line numbers when in lldb.